docker/Dockerfile.base

```diff
@@ -27,18 +28,25 @@ RUN apt-get -qq update \
     && rm -rf /var/lib/apt/lists/* /wheels \
     && (apt-get autoremove -y; apt-get autoclean -y)
 
+RUN pip3 install \
+    peewee \
+    zeroconf \
+    voluptuous
+
+COPY nginx/nginx.conf /etc/nginx/nginx.conf
+
 # get model and labels
-ARG MODEL_REFS=7064b94dd5b996189242320359dbab8b52c94a84
 COPY labelmap.txt /labelmap.txt
-RUN wget -q https://github.com/google-coral/edgetpu/raw/$MODEL_REFS/test_data/ssd_mobilenet_v2_coco_quant_postprocess_edgetpu.tflite -O /edgetpu_model.tflite
-RUN wget -q https://github.com/google-coral/edgetpu/raw/$MODEL_REFS/test_data/ssd_mobilenet_v2_coco_quant_postprocess.tflite -O /cpu_model.tflite
-
-RUN mkdir /cache /clips
+RUN wget -q https://github.com/google-coral/test_data/raw/master/ssdlite_mobiledet_coco_qat_postprocess_edgetpu.tflite -O /edgetpu_model.tflite
```

Comment:
Shouldn't the labelmap.txt file be updated from https://github.com/google-coral/test_data/raw/master/coco_labels.txt too?

There's this notable diff:

```
diff <(curl -s https://raw.githubusercontent.com/blakeblackshear/frigate/master/labelmap.txt) \
>    <(curl -s https://raw.githubusercontent.com/google-coral/test_data/master/coco_labels.txt)
8c8
< 7  car
---
> 7  truck
```

Reply:
That is intentional. The model will often flip flop between cars and trucks. By naming them the same, it prevents each label change from triggering a new object id to track.